changelog/@unreleased/pr-257.yml

```diff
@@ -0,0 +1,5 @@
+type: manual-task
```

Comment:
manual-task suggests that users will need to do something manually, however this shouldn't impact consumers :-)

```suggestion
type: improvement
```